Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -56,6 +62,24 @@ public void Initialize(TelemetryConfiguration configuration)
                         {                            
                             this.telemetryConfiguration = configuration;
 
+                            if (!this.DoNotParseHttp && configuration != null)
```

Comment:
>DoNotParseHttp [](start = 38, length = 14)

Why we create a new mechanizm for adding initializers? Why hidden one? Why not in config?